generators/app/index.js

```diff
@@ -48,9 +48,10 @@ class ReactNativeBootstrap extends Generator {
           'Drawer',
           'Crashlytics',
           'Push Notifications',
-          'Google Analytics',
+          'Firebase Analytics',
           'Redux Persist',
-          'Bitrise'
+          'Bitrise',
+          'Firebase Performance'
```

Comment:
Change these features on the bootstrap [README](https://github.com/Wolox/wolmo-bootstrap-react-native/blob/8030db265b57de3753852e5f2333ac5763c74b22/README.md) too.

Reply:
👍